src/translated_content/zh_CN/docs/developing-applications/building-blocks/actors/actors-overview.md

```
@@ -0,0 +1,132 @@
+
+当您的代码处理一条消息时，它可以向其他 actor 发送一条或多条消息，或创建新的 actor。底层运行时管理每个 actor 的运行方式、时间和位置，并在 actor 之间路由消息。
+
+大量的 actor 可以同时执行，并且 actor 彼此独立执行。
+
+## Dapr 中的 Actor 模型
+
+Dapr 包含一个专门实现[虚拟 Actor 模型](https://www.microsoft.com/research/project/orleans-virtual-actors/)的运行时。通过 Dapr 的实现，您可以根据 Actor 模型编写 Dapr actor，Dapr 利用底层平台提供的可扩展性和可靠性保证。
+
+每个 actor 都被定义为 actor 类型的一个实例，与对象是类的一个实例相同。例如，可能有一个实现计算器功能的 actor 类型，并且可能有许多此类型的 actor 分布在集群的各个节点上。每个这样的 actor 都由一个 actor ID 唯一标识。
+
+<img src="/images/actor_background_game_example.png" width=400 style="padding-bottom:25px;">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)